src/components/box.js

```diff
@@ -26,8 +26,12 @@
     const [opacity, setOpacity] = useState(opac);
     const [interactionBackground, setInteractionBackground] = useState('');
 
-    B.defineFunction('setBackgroundImage', url => {
-      setInteractionBackground(url);
+    B.defineFunction('setCustomBackgroundImage', url => {
+      setInteractionBackground(`url("${url}")`);
+    });
+
+    B.defineFunction('removeCustomBackgroundImage', () => {
+      setInteractionBackground(false);
```

Comment:
Keep types consistent for this state value, maybe reset it to empt6y string since that is the default